velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -93,7 +110,30 @@ class SubstraitVeloxPlanConverter {
     return lengths_;
   }
 
+  /// Used to find the function specification in the constructed function map.
+  std::string findFuncSpec(uint64_t id);
```

Comment:
naming: findFuncSpec -> findFunction

Mark the method 'const' and return const &.

Reply:
Fixed.